hearth/lib/hearth/http/fields.rb

```diff
@@ -61,6 +61,11 @@ def clear
         @entries = {}
       end
 
+      # @api private
+      def inspect
+        super.gsub(/ @entries={.*},/, '')
```

Comment:
Does the `inspect` get called on this for every log statement, even when the logger is a null logger or log level > debug?  (that is, we have a string passed to logger.debug, I assume that the string gets evaluated first, then passed to the function).  

I'm a little bit worried about the performance impact of this with frequent debug logs + large fields.  This could impact production performance with no way to disable it.

Reply:
Yeah, I think it's called every time, but actually, I think we can pass a block to the logger and it's lazy evaluated.

Reply:
Talked offline - we can tune this later.